.travis.yml

```diff
@@ -21,11 +22,19 @@ before_script:
 - go vet $EXCLUDE_VENDOR
 - misspell -error -locale US $EXCLUDE_VENDOR
 - staticcheck $EXCLUDE_VENDOR
-- if [[ "$TRAVIS_GO_VERSION" =~ 1.11 ]] && [ "$TRAVIS_TAG" != "" ]; then ./scripts/cross_compile.sh $TRAVIS_TAG; fi
 script:
 - go test -i $EXCLUDE_VENDOR
 - go test -run=TestNoRace $EXCLUDE_VENDOR
-- GOGC=10 go test -v -race -p=1 --failfast $EXCLUDE_VENDOR
-# - if [[ "$TRAVIS_GO_VERSION" =~ 1.11 ]]; then ./scripts/cov.sh TRAVIS; else GOGC=10 go test -v -race -p=1 --failfast $EXCLUDE_VENDOR; fi
-after_success:
-- if [[ "$TRAVIS_GO_VERSION" =~ 1.11 ]] && [ "$TRAVIS_TAG" != "" ]; then ghr --owner nats-io --token $GITHUB_TOKEN --draft --replace $TRAVIS_TAG pkg/; fi
+- if [[ "$TRAVIS_GO_VERSION" =~ 1.12 ]]; then ./scripts/cov.sh TRAVIS; else GOGC=10 go test -v -race -p=1 --failfast $EXCLUDE_VENDOR; fi
```

Comment:
We still need GOGC=10 and the -p=1?

Reply:
-p=1 for sure! (have enough of flaky tests right now without having servers started in parallel with port conflicts), for GOGC, not sure, going to check how much memory we use.

Reply:
Between server and test directories you mean?

Reply:
Yes. Some tests are still using config files that hardcode ports, etc..

Reply:
`test/` reaches more than 3GB with Go 1.12.. not able to pinpoint which test(s) are causing the bump (and likely should be moved to "no race" tests).

Reply:
Ok, test `TestQueueSubWeightOrderMultipleConnections` was using a lot. Moving 2 tests to the "no race" tests and will remove `GOGC=10`.